content/efcore/getting-started/samples/configuring-deletion/deletion-configuration/Program.cs

```diff
@@ -0,0 +1,30 @@
+using System;
+using System.Linq;
+using Microsoft.EntityFrameworkCore;
+
+public class Program
+{
+    public static void Main()
+    {
+        using (var context = new LibraryContext())
+        {
+            var reader = context.Readers.Include(r => r.Address).First();
+            var readers = context.Readers.First();
+            Console.WriteLine("{0} {1}", reader.FirstName, reader.LastName);
+            var address = context.Addresses.First();
+            Console.WriteLine("{0} {1}", address.City, address.State);
+
+            context.Readers.Remove(reader);
+            context.SaveChanges();
+
+            if (context.Addresses.Any(a => a.Id == 1))
+            {
+                Console.WriteLine("Address exists");
```

Comment:
The current example differentiates between `CascadeDelete` and the other two behaviors, but it doesn't differentiate between `SetNull` and `Restrict`. We need to also have some output for the `Address` foreign key (`Address.ReaderID`).